symengine/tests/basic/test_serialize-cereal.cpp

```diff
@@ -73,3 +76,16 @@ TEST_CASE("Test serialization using cereal", "[serialize-cereal]")
         real_mpfr(mpfr_class("0.35", 100, 10)));
 #endif
 }
+
+TEST_CASE("Test serialization exception", "[serialize-cereal]")
+{
+    RCP<const Basic> expr = se::parse("x + y");
+    std::string orig_data = expr->dumps();
+    std::vector<int> positions = {4, 8, 9, 15, 16};
```

Comment:
Can we have a comment here on how these positions were chosen? Random, or carefully calculated?